google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -374,18 +375,19 @@ private void validateConfiguration() throws ConfigurationException {
     if (!isValidConfigurationFile(getAppYamlPath())) {
       throw new ConfigurationException(
           GctBundle.getString("appengine.deployment.error.staging.yaml") + " "
-              + GctBundle.getString("appengine.deployment.error.staging.gotosettings"));
+              + GctBundle.getString("appengine.deployment.error.staging.file.gotosettings"));
     }
     try {
       if (isCustomRuntime()) {
-        if (StringUtils.isBlank(getDockerfilePath())) {
+        String dockerDirectoryPath = getDockerDirectoryPath();
+        if (StringUtils.isBlank(dockerDirectoryPath)) {
           throw new ConfigurationException(
-              GctBundle.message("appengine.flex.config.browse.dockerfile"));
+              GctBundle.message("appengine.flex.config.browse.docker.directory"));
         }
-        if (!isValidConfigurationFile(getDockerfilePath())) {
+        if (!isValidConfigurationFile(Paths.get(dockerDirectoryPath, DOCKERFILE).toString())) {
           throw new ConfigurationException(
               GctBundle.getString("appengine.deployment.error.staging.dockerfile") + " "
-                  + GctBundle.getString("appengine.deployment.error.staging.gotosettings"));
+                  + GctBundle.getString("appengine.deployment.error.staging.directory.gotosettings"));
```

Comment:
This message doesn't mention a Docker direction, just a "directory". Shouldn't it indicate that its the Docker directory we are talking about here?